data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/petrifies/petrifies_affect_allies.cfg

```diff
@@ -0,0 +1,165 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [attacks]affect_allies=
+##
+# Actions:
+# Give the side 1 leader a petrifies ability that only affects adjacent allies
```

Comment:
only empowers the weapons of adjacent allies

I'm wanting a clarification because I misread the text the first time as saying that it only petrifies adjacent allies (and in unit tests, it's possible for allied units to attack each other).

Reply:
Updated.